lib/publiccloud/azure.pm

```diff
@@ -249,24 +258,29 @@ sub generate_tags {
     my $definition = $self->generate_azure_image_definition();
 
 Generated the Azure Image name from the job settings. If present, it takes the PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION setting.
-If not present it generated the image definition name based on distri, version, flavor and SKU.
+If not present, it generates the image definition name based on distri, version, flavor and SKU.
 
+    input $basename: 'undefined/null/empty' to provide full image name;
+                     =1 (or not-null) to compose the base-name without 'sku', skipping the *_IMAGE_DEFINITION value.
```

Comment:
This is contra intuitive. Why don't you just decide based of the variable presence?

Reply:
I explained it in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19181#discussion_r1584531016